python/activator/visit.py

```diff
@@ -1,7 +1,6 @@
 __all__ = ["Visit"]
 
 from dataclasses import dataclass
-import lsst.geom
 
 
 @dataclass(frozen=True)
```

Comment:
Please add a comment warning that `Visit`'s members should be builtins, and must be both hashable and JSON-persistable. Otherwise we'll just make the same mistake again.